integration_tests/device.feature

```diff
@@ -2,39 +2,21 @@ Feature: Device Tests
 
 Scenario: API client can connect successfully to a load device and perform all operations
    Given redis container is started
-   And d3a is started using setup strategy_tests.external_devices (-t 1s -s 60m)
-   When the external client is started with test_load_connection
+   And d3a is started using setup strategy_tests.external_devices (-t 60s -s 60m -d 4h --slot-length-realtime 2s)
```

Comment:
Why are these changes necessary? Were there issues with the synchronization with d3a?

Reply:
Yes, at least this is what i suspect. I tried to eliminate the non-deterministic behaviour of the tests as well as making them shorter. They were only working before, because they were running for a long time. Eventually, they received a market_cycle event. But if you run the d3a only with `-d 4h` and without slowdown, the market_cycle event will receive the d3a-api-client only when the d3a is already finished. 
I hope this makes sense. Happy to explain more.